clang/include/clang/Basic/LangStandard.h

```diff
@@ -43,6 +43,7 @@ enum class Language : uint8_t {
   HLSL,
   ///@}
 };
+const char *LanguageToString(Language L);
```

Comment:
`languageToString`? This file correctly uses `functionName` unlike many other Clang files...